### PR TITLE
OSDOCS-4864: Updating CLI ref docs for 4.13

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-4864

Link to docs preview:
* https://57893--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/developer-cli-commands.html
* https://57893--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/administrator-cli-commands.html

QE review:
- [ ] QE has approved this change.

Additional information:

**These files are auto-generated, so I can't make any changes to the content in this PR. If anything needs to be fixed, it will need to be done in the source repo.**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
/label peer-review-needed

**NOTE: These files are auto-generated, so I can't make any changes to the content in this PR. If anything needs to be fixed, it will need to be done in the source repo.**

**Comment by bergerhoffer** [comment]
@zhouying7780 Can you please review these auto-generated CLI doc updates for 4.13?

Preview: 
* https://57893--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/developer-cli-commands.html
* https://57893--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/administrator-cli-commands.html

File: modules/oc-adm-by-example-content.adoc [hunk 1]
```diff
@@ -262,6 +262,18 @@ Collect debugging data for a given resource
 
 
 
+== oc adm migrate icsp
+Update imagecontentsourcepolicy file(s) to imagedigestmirrorset file(s).
```
Comment (mburke5678) [inline modules/oc-adm-by-example-content.adoc:266]: To align with [the docs](https://docs.openshift.com/container-platform/4.13/post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror-convert_post-install-preparing-for-users) on this:
s/imagecontentsourcepolicy/`ImageContentSourcePolicy` 
s/imagedigestmirrorset/`Image Digest Mirror Set`

File: modules/oc-by-example-content.adoc [hunk 2]
```diff
@@ -577,7 +577,7 @@ Display merged kubeconfig settings or a specified kubeconfig file
   # Show merged kubeconfig settings
   oc config view
   
-  # Show merged kubeconfig settings and raw certificate data
+  # Show merged kubeconfig settings and raw certificate data and exposed secrets
```
Comment (mburke5678) [inline modules/oc-by-example-content.adoc:580]: Should this be:
Show merged kubeconfig settings, raw certificate data, and exposed secrets
??

File: modules/oc-by-example-content.adoc [hunk 3]
```diff
@@ -1320,6 +1320,30 @@ Edit a resource on the server
...
+  # List recent events in the default namespace.
+  oc events
+  
+  # List recent events in all namespaces.
+  oc events --all-namespaces
+  
+  # List recent events for the specified pod, then wait for more events and list them as they arrive.
+  oc events --for pod/web-pod-13je7 --watch
+  
+  # List recent events in given format. Supported ones, apart from default, are json and yaml.
```
Comment (mburke5678) [inline modules/oc-by-example-content.adoc:1338]: s/Supported ones/Supported formats

**Comment by mburke5678** [comment]
@bergerhoffer A few nits (do we edit the cli command tect??). Otherwise LGTM

**Pull request merged**